### PR TITLE
Modify store REST API to return monetization data in tiers.

### PR DESCRIPTION
**Pull request by chamindias**

### CODE REVIEW

**Comment by malinthaprasan** [comment]
Let's check and fix Store React app as well

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml [hunk 1]
```diff
@@ -4344,8 +4344,27 @@ definitions:
...
+              properties:
+                fixedPrice:
+                  type: string
+                  example: "10"
+                pricePerRequest:
+                  type: string
+                  example: "1"
+                currencyType:
+                  type: string
+                  example: "LKR"
```
Comment (chamilaadhi) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml]: Shall we put USD

Comment (chamindias) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml]: https://github.com/wso2/carbon-apimgt/pull/6985/commits/d0f128d22d1412e1896864bde29ce15983ba11f0

**Comment by chamindias** [comment]
> Let's check and fix Store React app as well

https://github.com/wso2/carbon-apimgt/pull/6985/commits/1b3036a5234f149e30a66aaef1889181f809fa37

**Pull request merged**